### ISSUE TITLE
move towards non-racist terminology

### ISSUE THREAD

**Issue by mdik** [subgraph/oz#72]
Hi,

as discussed in other places
(e.g.
* https://english.stackexchange.com/questions/51088/alternative-term-to-blacklist-and-whitelist
* https://github.com/graphite-project/graphite-web/issues/1569
* https://github.com/whatwg/html/pull/281
)

it would be nice to replace the terms *blacklist* and *whitelist* by more suitable ones. Commonly used replacements are *blocklist* and *safelist*.

Sincerely,

Malte

**Comment by akuckartz** [comment]
:-1: Please research the etymology first. And as someone write on the linked page at Stackoverflow:
> This is an imaginary and puerile pseudo-controversy.

**Comment by timthelion** [comment]
mdik: Are you personally offended by the terms, or are you only worried that someone else might be? In the former case, I can re-assure you that the security community means no harm or offence by the use of these terms, though you may be right that their origins are unfortunately tainted with an awful history.

I am personally of the opinion that the term blacklist and whitelist are not of racial origin. If you've read the bible, or any other Abrahamic book, you may have noticed that the Indo-European symbolism of black and white actually comes from Genesis 1:1-1:2 "In the beginning, the earth was but a formless void. And GOD said, "let there be light" and there was light."

In this context, blackness refers to non-existance, where-as whiteness refers to existance. So when we use the term blacklist, we are listing the things that SHOULD NOT EXIST within the sandbox. And when we use the word whitelist, we are listing the things that MAY EXIST within the sandbox.

The fact that the words black and white have become racially polarized is really something that happened long after the symbolism of the colors came about.

Is Redhat a racist company just because evil genocidal European imperialists called the native Americans "redskins"?

**Comment by 0xBRM** [comment]
SJWs in charge of polluting issue trackers with much unneeded bikeshedding. Just fuck off. This is not an issue and it is not racially charged at all--YOU are the ones who, in their infinite paranoia, turn everything into a race issue. 
Like another poster said, the term "blacklist" comes from book bounded in black used by Henry VIII to listing monasteries to be dissolved.

@timthelion I would like to add that most native Americans died as a result of the black plague, smallpox, and the like, which can indeed be attributed to us, but not purposefully.

This is a fucking non-issue and you're spending developer time and patience in an attempt to appease to a group that you think should be protected. They do not need coddling, much like we don't. Stop pretending otherwise.

P.S.: Before you accuse me of racism, I would like to highlight the fact that I'm an actual Jew.

**Comment by timthelion** [comment]
"P.S.: Before you accuse me of racism, I would like to highlight the fact that I'm an actual Jew."

There is nothing more racist than to pull out the Jew card, as if merely being a Jew made you better, more pure, less guilty, than anyone else.

But I have a fealing, that these poor developers really didn't want to have this discussion on their github issues page. Can we move this to [r/mrasvssjws](https://www.reddit.com/r/mrasvssjws/)

**Comment by mdik** [comment]
Hi,

please close/wontfix if this is perceived as pollution/bikeshedding.

Thank you!

Malte

**Comment by CrisBRM** [comment]
@timthelion No, but it grants me immunity against antisemites like you due to the fact my people have been slaughtered for ages untold and face discrimination on a daily basis (you could serve as an example).

**Comment by cptfoo** [comment]
There are bad terms in Computer Science, like master/slave that are best just replaced earlier. Fighting over it is silly, and being of a specific ethnicity does not make it impossible for you to perpetuate racist culture.

**Comment by 0xBRM** [comment]
@cptfoo You're the one making it about race, as all liberals do. Slave and master do not have racial connotactions--the greek, the irish and the balkans were enslaved, as were Russian serfs. Blacklist denotes another thing in history with little do with blacks, you're the one who assumes it's racist due to your own biases. 

Kindly fuck off and go spend developer time elsewhere with your annoying SJW bikeshedding. You cannot be given a single inch.

**Comment by xSmurf** [comment]
Where's the pull request?

**Comment by DONNU-1-Shokolo** [comment]
it would be nice to replace the terms blacklist and whitelist by more suitable ones. Commonly used replacements are niggerlist and peoplelist.

**Comment by blacklistisnotracist** [comment]
I do agree blocklist/safelist may be better terms because they are more descriptive. That being said, calling blacklist or whitelist a racist word is simply wrong.

First, blacklist's origins have nothing to do with black people. What's more, it has never been used as an abusive word against black people. It's only a certain group of extreme non-black liberals who find the word offensive and try to make them look like it has racist connotations. It's as ridiculous as people who think wearing Chinese clothes is offensive for Chinese people unless they are worn by Chinese people.

Black, as a color, carries negative connotations across many cultures. It's so embedded that hunting down every word from every language is almost impossible and meaningless.

Let's stop calling a group of people black first. Let's start from there. Color comes first, race came later. Colors are tied to images, which can't be changed.

Let's stop the hypocrisy and focus on something that have real impacts.
